runtime-common/stdlib/msgpack/unpacker.h

```diff
@@ -25,6 +25,10 @@ class unpacker : private vk::not_copyable {
   const string& input_;
   std::size_t bytes_consumed_{0};
   msgpack::zone zone_;
+
+  // It's awful because unpacker and visitor are strongly logically connected,
+  // but there is no aggregation or composition. Should be fixed some day.
+  std::optional<vk::string_view> visitor_error_{std::nullopt};
```

Comment:
Let's make initialization of type std::optional consistent with `runtime-common/core/core-context.h:27`.